poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java

```diff
@@ -474,18 +466,21 @@ public void saveSensorAssetsToFlowgate(Map<String, Asset> exsitingSensorAssets,
       List<Sensor> sensors = null;
       int limit = 100;
       int offset = 0;
-      List<Asset> assetsNeedToSave = null;
+      List<Asset> newAssetsNeedToSave = null;
+      List<Asset> oldAssetsNeedToupdate = null;
       while ((sensors = client.getSensors(limit, offset)) != null) {
          if (sensors.isEmpty()) {
             logger.warn(
                   String.format("No sensor data from PowerIQ %s", client.getPowerIQServiceEndpoint()));
             break;
          }
-         assetsNeedToSave = new ArrayList<Asset>();
+         newAssetsNeedToSave = new ArrayList<Asset>();
+         oldAssetsNeedToupdate = new ArrayList<Asset>();
          for (Sensor sensor : sensors) {
             Asset asset = new Asset();
             Map<String,String> sensorMap = new HashMap<String,String>();
             sensorMap.put(FlowgateConstant.SENSOR_ID_FROM_POWERIQ, String.valueOf(sensor.getId()));
+            sensorMap.put(FlowgateConstant.POSITION_FROM_POWERIQ, sensor.getPosition());
```

Comment:
I'm thinking that position is not for powerIQ only. we may need this field for other system as well.

Reply:
Change to POSITION